ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { getMaximumGasTotalInHexWei } from '../../../../../shared/modules/gas.utils';
+import { GasLevelIconMap } from '../../../../helpers/constants/gas';
+import { PRIMARY } from '../../../../helpers/constants/common';
+import {
+  decGWEIToHexWEI,
+  decimalToHex,
+} from '../../../../helpers/utils/conversions.util';
+import { toHumanReadableTime } from '../../../../helpers/utils/util';
+import { useGasFeeContext } from '../../../../contexts/gasFee';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import I18nValue from '../../../ui/i18n-value';
+import InfoTooltip from '../../../ui/info-tooltip';
+import UserPreferencedCurrencyDisplay from '../../user-preferenced-currency-display';
+
+const EditGasItem = ({ estimateType, onClose }) => {
+  const {
+    estimateUsed,
+    gasFeeEstimates,
+    gasLimit,
+    setEstimateToUse,
+    updateTransaction,
+  } = useGasFeeContext();
+  const t = useI18nContext();
+
+  const { minWaitTimeEstimate, suggestedMaxFeePerGas } =
+    gasFeeEstimates[estimateType] || {};
+  const hexMaximumTransactionFee = suggestedMaxFeePerGas
+    ? getMaximumGasTotalInHexWei({
+        gasLimit: decimalToHex(gasLimit),
+        maxFeePerGas: decGWEIToHexWEI(suggestedMaxFeePerGas),
+      })
+    : null;
+
+  const onOptionSelect = () => {
+    setEstimateToUse(estimateType);
+    updateTransaction(estimateType);
+    onClose();
+  };
+
+  return (
+    <div
```

Comment:
If this is a clickable thing that does something, should this be an `<a>` or `<button>` instead? I realize this is debatable, and I'm not sure how we generally handle this sort of thing, just wondering if this would be more semantic?

Reply:
Yep I addressed it here: https://github.com/MetaMask/metamask-extension/pull/12719